api/types/types.go

```diff
@@ -24,3 +24,9 @@ type AuthResponse struct {
 	// Status is the authentication status
 	Status string `json:"Status"`
 }
+
+// POST /auth
+type ContainerWaitResponse struct {
+	// StatusCode is the status code of the wait job
+	StatusCode int `json:"StatusCode"`
```

Comment:
This json spec here is redundant since "StatusCode" would be marshalled into "SatusCode" for json already... but I see there are already a few places in this file where we are doing the same thing.